api/src/service/domain/workflow/system_information_eventsourcing.ts

```diff
@@ -0,0 +1,51 @@
+import { Ctx } from "../../../lib/ctx";
```

Comment:
why not using the structure of global_permissions_eventsourcing.ts - would be more especially regarding maintainability

Reply:
How? To get the provisioning status, we do not need to apply or validate events.

Reply:
Done